# specify robot track width in meters
track_width: 0.9925
# specify reduction ratio (70:1), if there is no put 1
reduction_ratio: 70
# specify reduction maximum linear velocity (m/s)
# max_vel: 0.95  # 60 m/min @3500rpm
max_rpm: 2600
max_vel_x: 0.7  # reduced speed for testing
max_vel_ang: 1.4
# specify robot wheel circumference (perimeter) in meters 
wheel_circumference: 1.13914122 # D:37cm
count_is_rpm: true
encoder_cpr: 6
encoder_min: -65536
encoder_max: 65536 
rate: 30 
enable_imu_yaw: true
imu_topic: $(arg imu_topic)
wrapping_enabled: false
wrp_lim: 0.3
sub_to_abs: false
imu_discarded: 100
tf_publish: false
odom_topic: /aristos/wheel/odom/feedback
tf_header_frame: $(arg prefix)odom
tf_child_frame: $(arg prefix)base_footprint